row/src/Row.ts

```diff
@@ -0,0 +1,6 @@
+import './Row.styles.css';
```

Comment:
Tbh I am not very confident with this technique for loading CSS for a webcomponent.
It will be added globally. What about using dead simple styles inside webcomponent like with lit ?

Reply:
fully agree
I'd go go Lit + Shadow DOM + `:host` selector for styling this